python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -776,6 +776,13 @@ class FgcmFitCycleConfig(pipeBase.PipelineTaskConfig,
         dtype=bool,
         default=False,
     )
+    fitCcdChromaticityDict = pexConfig.DictField(
+        doc="Per-band specification on whether to compute first-order quantum efficiency (QE) "
+            "adjustments. Any band not explicitly specified will default to False.",
+        keytype=str,
+        itemtype=bool,
+        default={},
+    )
```

Comment:
For various reasons, I have a bunch of these configs as key (band)/value.  When it's floating point values it makes sense, when it's boolean I guess it could be a list but I have this same format elsewhere so I'm sticking with it.  I've updated the docstring to be clearer.

Second, in terms of name, it gets translated in the utils `makeConfigDict`.  Answering that question, the proper use of camel case should allow all-caps acronyms (which I prefer and use in fgcm) but this is not allowed by DM developer guide.  It's my very minor form of protest.  😄 

Finally, this is meant in terms of fitting, but _not_ in terms of generating an updated LUT.  The fit parameters that come out of here can be used to make an updated LUT.  I'll explain better on the ticket.